usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -26,25 +27,227 @@ def handle(self, *args, **options):
         field_map = {
             "federal_action_obligation": "dollarsobligated",
             "description": "descriptionofcontractrequirement",
+            "other_statutory_authority": "otherstatutoryauthority",
             "modification_number": "modnumber"
         }
 
         value_map = {
             "data_source": "USA",
-            "award": lambda row: Award.objects.get_or_create(piid=row['piid'], type='C')[0],
-            "recipient": lambda row: LegalEntity.objects.get_or_create(recipient_name=row['dunsnumber'])[0],
-            "awarding_agency": lambda row: Agency.objects.get(subtier_code=self.get_agency_code(row['maj_agency_cat'])),
+            "award": lambda row: self.create_or_get_award(row),
+            "awarding_agency": lambda row: self.get_agency(row),
```

Comment:
I think everything coming from broker already conforms to the data structure we expect? It's worth looking into.